receiver/httpcheckreceiver/config_test.go

```diff
@@ -38,11 +38,22 @@ func TestValidate(t *testing.T) {
 				fmt.Errorf("%w: %s", errInvalidEndpoint, `parse "invalid://endpoint:  12efg": invalid port ":  12efg" after host`),
 			),
 		},
+		{
+			desc: "missing scheme",
+			cfg: &Config{
+				HTTPClientSettings: confighttp.HTTPClientSettings{
+					Endpoint: "www.opentelemetry.io/docs",
+				},
+			},
+			expectedErr: multierr.Combine(
+				fmt.Errorf("%w: %s", errInvalidEndpoint, `parse "www.opentelemetry.io/docs": invalid URI for request`),
+			),
+		},
 		{
 			desc: "valid config",
 			cfg: &Config{
 				HTTPClientSettings: confighttp.HTTPClientSettings{
-					Endpoint: "https://opentelemetry.io",
+					Endpoint: "https://opentelemetry.io:80/docs",
```

Comment:
Is this change necessary? I'm assuming `https://opentelemetry.io` is still a valid configuration

Reply:
Yes, it's valid. I considered adding a couple different tests for various URLs - with/without port, with/without path, but figured in the end we don't want to be testing the underlying functionality, so just settled on testing a "full-featured" URL. I can revert this change if you want, not even sure what value it brings :wink:

Reply:
It might be beneficial to have both one w/ the port & without. I know this is just testing the `ParseRequestURI` library under the hood, but i would feel better knowing there's a regression test in the case that someone replaces `ParseRequestURI` with a parsing library that requires the port in the future :D

Reply:
Changed so that we still have the previously existing test for `https://opentelemetry.io` endpoint and the `https://opentelemetry.io:80/docs` endpoint as a separate test.